from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(580, 440)
        self.centralWidget = QtWidgets.QWidget(MainWindow)
        self.centralWidget.setObjectName("centralWidget")
        self.label = QtWidgets.QLabel(self.centralWidget)
        self.label.setGeometry(QtCore.QRect(60, 190, 57, 16))
        self.label.setObjectName("label")
        self.label_1 = QtWidgets.QLabel(self.centralWidget)
        self.label_1.setGeometry(QtCore.QRect(250, 10, 221, 16))
        self.label_1.setObjectName("label_2")
        self.label_2 = QtWidgets.QLabel(self.centralWidget)
        self.label_2.setGeometry(QtCore.QRect(100, 30, 170, 16))
        self.label_2.setObjectName("label_2")
        self.label_1_1 = QtWidgets.QLabel(self.centralWidget)
        self.label_1_1.setGeometry(QtCore.QRect(250, 60, 221, 16))
        self.label_1_1.setObjectName("label_2")
        self.pushButton_model = QtWidgets.QPushButton(self.centralWidget)
        self.pushButton_model.setGeometry(QtCore.QRect(200, 180, 161, 32))
        self.pushButton_model.setObjectName("pushButton_model")
        self.pushButton_graph = QtWidgets.QPushButton(self.centralWidget)
        self.pushButton_graph.setGeometry(QtCore.QRect(220, 405, 141, 32))
        self.pushButton_graph.setObjectName("pushButton_graph")
        self.label_3 = QtWidgets.QLabel(self.centralWidget)
        self.label_3.setGeometry(QtCore.QRect(90, 80, 221, 16))
        self.label_3.setObjectName("label_3")
        self.label_4 = QtWidgets.QLabel(self.centralWidget)
        self.label_4.setGeometry(QtCore.QRect(40, 110, 271, 16))
        self.label_4.setObjectName("label_4")
        self.label_5 = QtWidgets.QLabel(self.centralWidget)
        self.label_5.setGeometry(QtCore.QRect(160, 150, 151, 16))
        self.label_5.setObjectName("label_5")
        self.spinbox_intensivity_gen = QtWidgets.QDoubleSpinBox(self.centralWidget)
        self.spinbox_intensivity_gen.setGeometry(QtCore.QRect(320, 30, 141, 22))
        self.spinbox_intensivity_gen.setMaximum(9999.99)
        self.spinbox_intensivity_gen.setObjectName("spinbox_intensivity_gen")
        self.spinbox_intensivity_oa = QtWidgets.QDoubleSpinBox(self.centralWidget)
        self.spinbox_intensivity_oa.setGeometry(QtCore.QRect(320, 80, 141, 22))
        self.spinbox_intensivity_oa.setMaximum(99999.99)
        self.spinbox_intensivity_oa.setObjectName("spinbox_intensivity_oa")
        self.spinbox_intensivity_oa_range = QtWidgets.QDoubleSpinBox(self.centralWidget)
        self.spinbox_intensivity_oa_range.setGeometry(QtCore.QRect(320, 110, 141, 22))
        self.spinbox_intensivity_oa_range.setMaximum(99999.99)
        self.spinbox_intensivity_oa_range.setObjectName("spinbox_intensivity_oa_range")
        self.spinbox_time_model = QtWidgets.QDoubleSpinBox(self.centralWidget)
        self.spinbox_time_model.setGeometry(QtCore.QRect(320, 150, 141, 22))
        self.spinbox_time_model.setMinimum(1.0)
        self.spinbox_time_model.setMaximum(99999999.99)
        self.spinbox_time_model.setObjectName("spinbox_time_model")
        self.tableWidget = QtWidgets.QTableWidget(self.centralWidget)
        self.tableWidget.setGeometry(QtCore.QRect(50, 210, 501, 192))
        self.tableWidget.setObjectName("tableWidget")
        self.tableWidget.setColumnCount(2)
        self.tableWidget.setRowCount(4)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(1, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(2, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setVerticalHeaderItem(3, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setHorizontalHeaderItem(0, item)
        item = QtWidgets.QTableWidgetItem()
        self.tableWidget.setHorizontalHeaderItem(1, item)
        MainWindow.setCentralWidget(self.centralWidget)

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "Лабораторная работа №1"))
        self.label.setText(_translate("MainWindow", ""))
        self.label_1.setText(_translate("MainWindow", "Поступление заявок"))
        self.label_1_1.setText(_translate("MainWindow", "Обработка заявок"))
        self.label_2.setText(_translate("MainWindow", "Интенсивность:"))
        self.pushButton_model.setText(_translate("MainWindow", "Смоделировать СМО"))
        self.pushButton_graph.setText(_translate("MainWindow", "Построить график"))
        self.label_3.setText(_translate("MainWindow", "Интенсивность:"))
        self.label_4.setText(_translate("MainWindow", "Разброс интенсивности:"))
        self.label_5.setText(_translate("MainWindow", "Время моделирования:"))
        item = self.tableWidget.verticalHeaderItem(0)
        item.setText(_translate("MainWindow", "Загруженность системы"))
        item = self.tableWidget.verticalHeaderItem(1)
        item.setText(_translate("MainWindow", "Среднее время пребывания заявки в СМО"))
        item = self.tableWidget.verticalHeaderItem(2)
        item.setText(_translate("MainWindow", "Обработанные заявки"))
        item = self.tableWidget.verticalHeaderItem(3)
        item.setText(_translate("MainWindow", "Общее время моделирования"))
        item = self.tableWidget.horizontalHeaderItem(0)
        item.setText(_translate("MainWindow", "Теоретическая"))
        item = self.tableWidget.horizontalHeaderItem(1)
        item.setText(_translate("MainWindow", "Фактическая"))
